src/main/java/com/jmb/springfactory/controller/api/UserController.java

```diff
@@ -4,6 +4,7 @@
 
 import static org.apache.commons.lang3.StringUtils.isNotBlank;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.Authentication;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.springframework.security.core.Authentication'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)